onnxscript/optimizer/_function_folding_test.py

```diff
@@ -60,13 +60,11 @@ def test_sequence_concat(self):
         """
         )
         optimized = optimizer.optimize(
-            model,
-            onnx_shape_inference=False,
-            num_iterations=1,
+            model, onnx_shape_inference=False, num_iterations=1, inline=False
```

Comment:
Line [58] above should be changed to `z = ConcatFromSequence <axis=0> (t3)` ... the required attribute axis was missing in test-case (ignored and optimized by old optimizer, not by new).

Reply:
I merged in the changes I made while debugging these ... hope that's fine.

Reply:
Thanks!